azure-pipelines.yml

```
@@ -3,7 +3,7 @@ resources:
   - repository: wire-ios-shared-resources
     type: github
     name: wireapp/wire-ios-shared-resources
-    ref: refs/heads/master # Branch to fetch the jobs template from
+    ref: refs/heads/xcode10_1 # Branch to fetch the jobs template from
```

I updated `wire-ios-shared-resources` so this can be reverted now

```suggestion
    ref: refs/heads/master # Branch to fetch the jobs template from
```